snapcraft/cli/store.py

```diff
@@ -17,6 +17,7 @@
 import sys
 from textwrap import dedent
 from typing import TextIO
+from snapcraft.formatting_utils import humanize_list
```

Comment:
I'd rather see `from snapcraft import formatting_utils` rather than the function itself.